### DISCUSSION TITLE
Just Chilling Man! Fumadocs are working great.

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#901] [General]
Hey @fuma-nama, the docs are working great and there is no issue currently. I am free to contribute to fumadocs, is there any help required in designing or updating ? I can help you. 

Thanks for this great docs, I would be facing so many problems now if I was using nextra or docusaurus, but there are none with fumadocs. 

### Happy Coding.

**Comment by Gitstar-OC** [comment]
Are there any docs on blogs ? Like there is just this one thing about blog "https://fumadocs.vercel.app/docs/ui#blog-sites" are you planning to make improvements to blog and develop it like https://ghost.org. 

Like adding docs about setting up blog section and styling in it ? Should I help with that and styling

**Reply by fuma-nama** [reply]
Blog is not our main goal (as suggested by the name `fumadocs` and all docs automation tools we provided). I'm working actively with Next.js and some libraries we rely on (e.g. Shiki) to ensure Next.js 15 + Fumadocs 14 will work out-of-the-box for everyone.

**Reply by fuma-nama** [reply]
There're many issues created due to Next.js changing `params` and `searchParams` into async, and it's not all backward compatible which requires reporting bugs and waiting for upstream fixes

**Reply by Gitstar-OC** [reply]
Alright, I will check the issues and try to fix those.

**Reply by fuma-nama** [reply]
If you want to help, welcome to check the feature requests. I have approved some of them

**Reply by StevenRotelli** [reply]
Should one avoid blogs when using the `content-collections` createMDXSource? docs is working great, but mixing the two seems a bit tricky.

**Reply by fuma-nama** [reply]
You don't need it, it's a Fumadocs adapter
